lib/src/model.dart

```diff
@@ -507,7 +507,7 @@ class Library extends ModelElement {
 
   String get path => _library.definingCompilationUnit.name;
 
-  String get nameForFile => name.replaceAll(':', '-');
```

Comment:
changing this to `dirName` and adding a `fileName` that returns `libName-library.html` to make this clear.